resources/src/offer.rs

```diff
@@ -0,0 +1,88 @@
+use deserialize;
+use asset::PartialAsset;
+
+/// The ratio between the asking and selling price
+#[derive(Serialize, Deserialize, Debug)]
+pub struct PriceRatio {
+    #[serde(rename = "n")] numerator: u64,
+    #[serde(rename = "d")] denominator: u64,
+}
+
+/// An offer being made for particular assets at a particular exchange rate.
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Offer {
+    id: i64,
+    paging_token: String,
+    seller: String,
+    selling: PartialAsset,
+    buying: PartialAsset,
+    #[serde(deserialize_with = "deserialize::amount")] amount: i64,
+    #[serde(rename = "price_r")] price_ratio: PriceRatio,
+    #[serde(deserialize_with = "deserialize::amount")] price: i64,
```

Comment:
I think both price and amount should be floats otherwise you could only trade whole units of token